src/CreateHandler/CreateHandler.php

```diff
@@ -37,10 +37,11 @@ public function __construct(string $classSkeleton = self::CLASS_SKELETON, string
     }
 
     /**
+     * @param string $class
```

Comment:
Please undo all these changes as they are not needed. The method declaration already contains this information!
Additionally the topic of this PR is the fix of #82 rather than adding more DocBlocks or clean up anything.

That makes the review easier. Thanks!